core/domain/state_domain.py

```diff
@@ -798,6 +798,7 @@ class WrittenTranslations(python_utils.OBJECT):
 
     def __init__(self, translations_mapping):
         """Initializes a WrittenTranslations domain object."""
```

Comment:
Where's the doc for `translations_mapping`?

Reply:
Done